tinytroupe/experimentation/__init__.py

```
@@ -0,0 +1,11 @@
+
+import logging
+logger = logging.getLogger("tinytroupe")
+
+###########################################################################
+# Exposed API
+###########################################################################
+from .randomization import ABRandomizer
+from .proposition import Proposition, check_proposition
```

Ruff (0.8.2)

`.proposition.check_proposition` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)